terraform/autospotting/variables.tf

```diff
@@ -46,6 +46,10 @@ variable "autospotting_regions_enabled" {
   description = "Regions that autospotting is watching"
 }
 
+variable "autospotting_tag_filters" {
+  description = "tags to filter which ASGs autospotting considers"
+}
```

Comment:
I would improve a bit the description (both here and in the module) to let the user know it supports both comma & space separation, and that is a list. A bit like for asg_disallowed_instance_types, I would also add the default value to spot-enable=true
Like you did in Cloudformation :)